src/metalperformanceshaders.cs

```diff
@@ -788,6 +788,9 @@ interface MPSCnnNeuronLinear {
 		[Export ("b")]
 		float B { get; }
 
+		[Deprecated (PlatformName.TvOS, 12, 0)]
+		[Deprecated (PlatformName.iOS, 12, 0)]
+		[Deprecated (PlatformName.MacOSX, 10, 14)]
```

Comment:
no alternative are mentioned ?

Reply:
Not that I am aware of, also nothing on docs https://developer.apple.com/documentation/metalperformanceshaders/mpscnnneuronlinear/1648888-initwithdevice?language=objc that said I still need go through the `MPSNeuralNetwork.framewrok` so it might change once I review the diff.